test/Broadway/CommandHandling/ClosureCommandHandlerTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+
+namespace Broadway\CommandHandling;
+
+use Broadway\TestCase;
+
+class ClosureCommandHandlerTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function it_delegates_command_to_proper_handle_function()
+    {
+        $commandHandler = new TestClosureCommandHandler();
+        $command        = new ClosureCommandHandlerTestCommand();
+        $commandHandler->handle($command);
+
+        $this->assertTrue($commandHandler->handled);
```

Comment:
the handled property does not exist in the ClosureCommandHandler (or the CommandHandlerInterface)